client/state/data-layer/wpcom/sites/activity/from-api.js

```diff
@@ -28,6 +28,55 @@ export function transformer( apiResponse ) {
 	return map( orderedItems, processItem );
 }
 
+const getActivityDescription = item => {
+	const { generator, object } = item;
+
+	switch ( item.name ) {
+		case 'post__published':
+			return (
+				`<a href="/read/blogs/${ generator.blog_id }/posts/` +
+				`${ object.object_id }">${ object.name }</a>` +
+				'<div class="activity-log-item__event">Post published</div>'
```

Comment:
I think these have to be translatable?

Reply:
yeah definitely. this PR should not be merged or considered final/mergable